docs/api/app.md

```diff
@@ -208,6 +208,15 @@ This method guarantees that all `beforeunload` and `unload` event handlers are
 correctly executed. It is possible that a window cancels the quitting by
 returning `false` in the `beforeunload` event handler.
 
+### `app.exit(exitCode)`
+
+* `exitCode` Integer
+
+Exits immediately with `exitCode`.
+
+All windows will be closed immediately without asking user and the `before-quit`
+and `will-quit` events will not be emitted.
```

Comment:
It looks like `will-quit` is still fired on `0.35.4`:

```js
require('electron').app.on('will-quit', function () {
  console.log('will-quit')
})
require('electron').app.on('ready', function () {
  require('electron').app.exit(0);
}, 1000)
```

Produces:

```
will-quit
```

when run. Want me to create a new issue for this?